src/pages/companion/wrapped/companySummary.tsx

```diff
@@ -0,0 +1,129 @@
+"use client";
+
+import NavBarContainer from "@/components/companion/navigation/NavBarContainer";
+import Image from "next/image";
+import { useState } from "react";
+import { useRouter } from "next/navigation"
+import { motion, useMotionValue, useTransform, animate, AnimatePresence } from 'framer-motion';
+
+interface TopCompaniesProps {
+  isPartner: boolean;
+}
+
+// Combined stats between amazon and aws, as well as google and google cloud
+const topCompanies = [
+  {
+    rank: 1,
+    name: "Amazon",
+    taps: 107,
+    logo: "https://upload.wikimedia.org/wikipedia/commons/a/a9/Amazon_logo.svg",
+  },
+  {
+    rank: 2,
+    name: "Electronic Arts",
+    taps: 64,
+    logo: "https://upload.wikimedia.org/wikipedia/commons/0/0d/Electronic-Arts-Logo.svg",
+  },
+  {
+    rank: 3,
+    name: "Google",
+    taps: 59,
+    logo: "https://upload.wikimedia.org/wikipedia/commons/2/2f/Google_2015_logo.svg",
+  },
+  {
+    rank: 4,
+    name: "Meta",
+    taps: 57,
+    logo: "https://upload.wikimedia.org/wikipedia/commons/a/ab/Meta-Logo.png",
+  },
+  {
+    rank: 5,
+    name: "SAP",
+    taps: 44,
+    logo: "https://upload.wikimedia.org/wikipedia/commons/5/59/SAP_2011_logo.svg",
+  },
+];
```

Comment:
Instead of Wikimedia photos, could we update these to use the same profile pictures we have on Company Profiles for consistency?